index.bs

```diff
@@ -394,7 +394,9 @@ No information from the underlying platform is exposed directly. An execution ti
 
 Note: The group is <a href="https://github.com/webmachinelearning/webnn/issues/85">soliciting further input</a> on the proposed execution time analysis fingerprinting vector and will augment this section with more information and mitigations to inform the implementers of this API.
 
-Implementers of this API are expected to be familiar with the <a href="https://gpuweb.github.io/gpuweb/#security-privacy">WebGPU Privacy Considerations</a>.
+This API mitigates against timing attacks that rely on data caching or persistence by not intrinsically supporting custom shader authoring. Instead, the API builds upon pre-existing shaders and lower level primitives of the browser or the underlying OS. Web developers who interface with {{GPUDevice}} are expected to be aware of <a href="https://gpuweb.github.io/gpuweb/#privacy-user-agent-state">WebGPU compilation cache considerations</a>.
```

Comment:
I think this sentence may be a bit backward. Should we say "Unlike APIs like WebGL, and WebGPU; this API does not intrinsically supporting custom shader authoring; and as a result is not prone to timing attacks that rely on shader caches, or other persistent data. "?